content/_code-samples/multisigning/py/multisigning.py

```diff
@@ -0,0 +1,75 @@
+"""Example of how we can multisign a transaction"""
+from xrpl.clients import JsonRpcClient
+from xrpl.models.requests import SubmitMultisigned
+from xrpl.models.transactions import AccountSet, SignerEntry, SignerListSet
+from xrpl.transaction import (
+    autofill,
+    autofill_and_sign,
+    multisign,
+    send_reliable_submission,
+    sign,
+)
+from xrpl.utils import str_to_hex
+from xrpl.wallet import generate_faucet_wallet
+
+client = JsonRpcClient("https://s.altnet.rippletest.net:51234")
+
```

Comment:
Please add a comment saying that "This will only work with version `2.0.0-beta.0` or later" (It uses the `multisign` function which is only implemented in the beta brach currently.